README.md

```diff
@@ -66,21 +66,21 @@ We have a [wiki](https://github.com/gfx-rs/wgpu/wiki) that serves as a knowledge
 
 ## Supported Platforms
 
-   API   |    Windows                    |  Linux & Android          |    macOS & iOS      |
-  -----  | ----------------------------- | ------------------------- | ------------------- |
-  Vulkan | :white_check_mark:            | :white_check_mark:        |                     |
-  Metal  |                               |                           | :white_check_mark:  |
-  DX12   | :white_check_mark: (W10 only) |                           |                     |
-  DX11   | :hammer_and_wrench:           |                           |                     |
-  GLES3  |                               | :ok:                      |                     |
-  Angle  | :ok:                          | :ok:                      | :hammer_and_wrench: |
+   API   |    Windows                         |  Linux & Android          |    macOS & iOS      |
+  -----  | ---------------------------------- | ------------------------- | ------------------- |
+  Vulkan | :white_check_mark:                 | :white_check_mark:        |                     |
+  Metal  |                                    |                           | :white_check_mark:  |
+  DX12   | :white_check_mark: (W10 & 11 only) |                           |                     |
```

Comment:
Could we do `(W10+)` to save some characters and be future proof.

Reply:
Yeah, sure